### PR TITLE
Add String.parse* functions

### PR DESCRIPTION
**Pull request by aswan**

### CODE REVIEW

File: expr/functions.go [hunk 1]
```diff
@@ -239,3 +245,48 @@ func mathSqrt(args []zngnative.Value) (zngnative.Value, error) {
 
 	return zngnative.Value{zng.TypeFloat64, r}, nil
 }
+
+func stringParseInt(args []zngnative.Value) (zngnative.Value, error) {
+	switch args[0].Type.ID() {
+	case zng.IdString, zng.IdBstring:
+		i, perr := strconv.ParseInt(args[0].Value.(string), 10, 64)
```
Comment (henridf) [inline expr/functions.go]:
```suggestion
		i, perr := strconv.Atoi(args[0].Value.(string))
```

Comment (henridf) [reply expr/functions.go]: nitty

**Comment by henridf** [comment]
Not specific to this PR, but these functions will get added to the docs somewhere right?

**Comment by aswan** [comment]
> Not specific to this PR, but these functions will get added to the docs somewhere right?

Yes, but we haven't documented expression syntax at all yet, in my head I have documenting functions lumped in with that...

**Pull request merged**